pkg/scaling/modifiers/formula.go

```diff
@@ -53,6 +56,16 @@ func HandleScalingModifiers(so *kedav1alpha1.ScaledObject, metrics []external_me
 			log.Error(err, "error applying custom scalingModifiers.Formula")
 		}
 		log.V(1).Info("returned metrics after formula is applied", "metrics", metrics)
+	} else if fallbackMetrics != nil {
+		var handledModifierMetrics []external_metrics.ExternalMetricValue
+		for _, metric := range fallbackMetrics {
+			var ret external_metrics.ExternalMetricValue
+			ret.MetricName = kedav1alpha1.CompositeMetricName
```

Comment:
In the normal logic path the program would call 'applyScalingModifiersFormula()', which created new metrics with a 'CompositeMetricName'. However, prior to my fixing the code, in the fallback logic path, the program returned directly, so the metrics still had their own individual metric names. Therefore, I updated the metric names to be consistent across both paths.

Reply:
My question is because in `CompositeMetricName` we should respond a single value instead of all the values in fallback. I mean, fallback should ignore the formula and returns the desired value currently

Reply:
I don't see the use case where we need to override all the metrics names if we correct the value directly on `doFallback`

Reply:
`doFallback` currently doesn't modify the metric name. I can move this code to `doFallback`. I considered this function to be responsible for handling all the modifier scaler conditions, so I placed it here. Additionally, the fallback metric should only be a single metric, right? Right now, I am returning all of them, which results in more than one fallback metric being returned to HPA.

Reply:
> Additionally, the fallback metric should only be a single metric, right?

Yes, we should return a single metric for a given HPA request, and the HPA Controller will request all the metrics one by one, but I think that I was wrong and at this point, probably you have a single metric (we use slices to fulfil the signature, but usually we return a single metric)

Reply:
I am thinking than maybe two metrics will be fallback at the same time so I update to return only one fallback metric.